command/agent/template/template.go

```diff
@@ -191,6 +199,17 @@ func (ts *Server) Run(ctx context.Context, incoming chan string, templates []*ct
 				return fmt.Errorf("template server: %w", err)
 			}
 
+			// Calculate the amount of time to backoff using exponential backoff
+			sleep, err := restartBackoff.Next()
```

Comment:
Using "next" here in stead of NextSleep so we can log how long it's sleeping for.